beancount_import/source/schwab_csv.py

```diff
@@ -33,6 +33,7 @@
     2015-11-09 open Assets:Investments:Schwab:Brokerage-1234
          schwab_account: "Brokerage XXXX-1234"
          div_income_account: "Income:Dividend:Schwab"
+         interest_income_account: "Income:Interest:Schwab"
```

Comment:
Oh, minor nit but can you also revise the prose sentence immediately preceding this example to read "... for recording dividends, interest, capital gains, fees, and taxes." (Addition of "interest," in the list.)